examples/docs_example/src/osdkExample.tsx

```diff
@@ -0,0 +1,155 @@
+import { createClient } from "@osdk/client";
+import {
+  Employee,
+  Office,
+  Ontology,
+  promoteEmployee,
+  promoteEmployeeObject,
+} from "./generatedNoCheck2";
+
+export const client = createClient(
+  Ontology,
+  "",
+  async () => {
+    return "";
+  },
+);
+
+export async function osdkObjectSetExample() {
+  /**
+   * OBJECT SETS
+   */
+
+  // Simple where clause
+  // This defaults to AND together all the parameters
+  const simpleFilteredEmployeeObjectSet = await client(Employee).where({
+    fullName: { $contains: "Clooney" },
+    employeeId: { $eq: 12 },
+  });
+
+  // Where clause with ors, ands, nots
+  const complexFilteredEmployeeObjectSet = await client(Employee).where({
+    $or: [{ fullName: { $contains: "Clooney" }, startDate: { $gt: 10 } }, {
+      $and: [{ $not: { fullName: { $contains: "Pitt" } } }, {
+        $or: [{ fullName: { $contains: "Downey" } }, {
+          fullName: { $contains: "Hemsworth" },
+          employeeId: { $gte: 20 },
+        }],
+      }],
+    }],
+  });
+
+  // Where clause GEOTYPES
+
+  // Within clauses take either a polygon or bounding box
+  const withinFilteredObjectSet = await client(Office).where({
+    $or: [
+      {
+        entrance: { $within: { distance: [1_000, "miles"], of: [0, 0] } },
+      },
+      { entrance: { $within: { bbox: [0, 1, 2, 3] } } },
+      { entrance: { $within: [0, 1, 2, 3] } },
+      {
+        entrance: { $within: { polygon: [[[0, 1], [0, 0], [1, 1], [1, 0]]] } },
```

Comment:
Hrm is this the actual syntax? there should be a 5th entry here for `[0,1]` again.
https://datatracker.ietf.org/doc/html/rfc7946#section-3.1.6

(Same for the rest of the file everywhere)

Reply:
Ah wait you're right good flag, yea the points I gave would be invalid GeoJSON since I didn't close the loop